README.md

```diff
@@ -306,6 +306,32 @@ from prometheus_client import start_wsgi_server
 start_wsgi_server(8000)
 ```
 
+#### ASGI
+
+To use Prometheus with [ASGI](http://asgi.readthedocs.org/en/latest/), there is
+`make_asgi_app` which creates an ASGI application.
+
+Save the snippet below in a `myapp.py` file
+
+```python
+from prometheus_client import make_asgi_app
+
+app = make_asgi_app()
+```
+Such an application can be useful when integrating Prometheus metrics with ASGI
+apps.
+
+The app can be used to serve the metrics through an ASGI implementation, such
```

Comment:
This seems unnecessary. These are the client_python docs, not an ASGI intro.

Reply:
I simplified the documentation.